backend/app/deps/authorization_deps.py

```diff
@@ -52,7 +51,7 @@ async def get_role(
         return RoleType.OWNER
 
     authorization = await AuthorizationDB.find_one(
-        AuthorizationDB.dataset_id == PyObjectId(dataset_id),
```

Comment:
Yeah for some reason i have to use pydanticObjectId instead of pyObjectId from now on. Not sure why though.
I researched a bit the internet told me they are basically identical.